index.js

```diff
@@ -116,6 +121,15 @@ async function doStuff() {
             issue_number: issue.number,
             state: 'closed',
         });
+        const availableColumns = await octokit.rest.projects.listColumns({
+            project_id: getProjectIdFromUrl(github.context.payload.project_card.project_url),
```

Comment:
This is kinda disappointing, but didn't find a direct link 😢

Reply:
This is a REST API, so you're supposed to get that `project_url` to get the project id... From what I can tell octokit only offers https://octokit.github.io/rest.js/v18#custom-requests for that.

Reply:
Just remembered where the similar logic in the monolith was: https://github.com/mozillach/gh-projects-content-queue/blob/master/lib/column.js#L31